src/services/utilities.ts

```diff
@@ -4042,14 +4042,12 @@ function getSymbolParentOrFail(symbol: Symbol) {
     return Debug.checkDefined(
         symbol.parent,
         `Symbol parent was undefined. Flags: ${Debug.formatSymbolFlags(symbol.flags)}. ` +
-            `Declarations: ${
-                symbol.declarations?.map(d => {
-                    const kind = Debug.formatSyntaxKind(d.kind);
-                    const inJS = isInJSFile(d);
-                    const { expression } = d as any;
-                    return (inJS ? "[JS]" : "") + kind + (expression ? ` (expression: ${Debug.formatSyntaxKind(expression.kind)})` : "");
-                }).join(", ")
-            }.`,
+            `Declarations: ${symbol.declarations?.map(d => {
+                const kind = Debug.formatSyntaxKind(d.kind);
+                const inJS = isInJSFile(d);
+                const { expression } = d as any;
+                return (inJS ? "[JS]" : "") + kind + (expression ? ` (expression: ${Debug.formatSyntaxKind(expression.kind)})` : "");
+            }).join(", ")}.`,

```

Comment:
I looked into it. Essentially swc changed its representation of optional chaining to be more like babel and I think that change in the AST caused this change where it will try to be more "inline". It seems in this scenario, changing the call expr to have an optional chain in it will now causes different behaviour unlike before. I will fix this tomorrow.

Reply:
By the way, something I've wanted to do for a while is have certain codebases in a test suite in order to catch regressions before a release. I haven't gotten around to it yet, but when I add it then I will add typescript's repo https://github.com/dprint/dprint-plugin-typescript/issues/13

Reply:
@jakebailey I fixed it now and did a release.

Reply:
Thank you!!